### PR TITLE
Feat: windows build support, more readme instructions

### PR DESCRIPTION
**Pull request by TheyCallMeCheng**
I added support for windows by adding another script in package.json, this error popped up when trying to build:

image

I added some instructions in the readme so it's easier for new people to start contributing.

### CODE REVIEW

**Comment by TheyCallMeCheng** [comment]
> works for me in windows though, @nexxeln thoughts?

Are you using windows WSL w/ ubuntu maybe? 
rm (which is used by npm build script) shouldn't work on windows, or maybe I'm doing something wrong

File: README.md [hunk 1]
```diff
@@ -18,6 +18,21 @@ yarn create t3-app
 # or
 pnpm create t3-app@latest
 ```
+To start your app 
+```bash
+yarn
+yarn dev
+```
```
Comment (nexxeln) [inline README.md:26]: Please remove this section

**Review by nexxeln — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -19,6 +19,33 @@ yarn create t3-app
...
+
+To start the package locally on mac/linux
+```bash
+npm start
+```
+
+To start the package on windows:
+```bash
+npm start-w
+```
```
Comment (nexxeln) [inline README.md:47]: To start the package on windows:
```bash
pnpm start-w
```

**Review by nexxeln — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TheyCallMeCheng) [reply README.md]: Resolved in 44fd30a35613cea5114750ee12218241712c7c13

**Review by nexxeln — APPROVED** [review]

**Pull request merged**